docs/examples/offloading-multiple-pods.md

```diff
@@ -0,0 +1,251 @@
+
+```bash
+kubectl get namespaces liqo-demo --kubeconfig="$KUBECONFIG_DESTINATION1"
+kubectl get namespaces liqo-demo --kubeconfig="$KUBECONFIG_DESTINATION2"
+```
+
+Both the commands should return the following output:
+
+```text
+NAME        STATUS   AGE
+liqo-demo   Active   70s
+```
+
+## Deploy applications
+
+Now it's time to deploy the application.
+
+In order to create a replica of the application in every *destination* cluster is necessary to enforce these conditions.
+
+* The *deployment* resource must produce at least one *pod* for each *destination* cluster.
+* Each *destination* cluster must schedule at most one *pod* on its nodes.
+
+To get this result you can exploit these *kubernetes* features:
+
+* Set a number of replicas in the *deployment* which is equal to the number of *destination* clusters
+* Set *topologySpreadConstraints* inside the *deployment*'s template, which sets the *maxSkew* equal to **1**.
+
+The file `manifests/deployment.yaml` contains an example of a *deployment* which satisfies these conditions.
+Let's deploy it:
+
+```bash
+kubectl apply -f ./manifests/deployment.yaml -n liqo-demo
+```
+
+```{admonition} More replicas
+If the deployment uses a number of replicas which is higher than the number of *virtual nodes*, the pods will be scheduled respecting the *maxSkew* value, which guarantees that the difference between the maximum number of pods (scheduled on a single node) and the minimum will be **1**
+```
+
+Checking the pod status, it is possible to verify that one has been scheduled onto the *destination1* cluster, and the other on *destination2*, and it is correctly running.
+Every *destination* cluster has scheduled one *pod* on its nodes.
+
+```bash
+kubectl get pod -n liqo-demo -o wide
+```
+
+```text
+NAME                            READY   STATUS    RESTARTS   AGE     IP            NODE                NOMINATED NODE   READINESS GATES
+liqo-demo-app-777fb9fc8-bbt4d   1/1     Running   0          7m28s   10.113.0.65   liqo-destination1   <none>           <none>
+liqo-demo-app-777fb9fc8-wrjph   1/1     Running   0          7m28s   10.109.0.62   liqo-destination2   <none>           <none>
+```
+
+## Tear down the playground
+
+### Unoffload namespaces
+
+Before starting the uninstallation process, make sure that all namespaces are unoffloaded:
+
+```bash
+liqoctl unoffload namespace liqo-demo
+```
+
+### Revoke peerings
+
+Similarly, make sure that all the peerings are revoked:
+
+```bash
+liqoctl unpeer in-band --remote-kubeconfig "$KUBECONFIG_DESTINATION1" 
+liqoctl unpeer in-band --remote-kubeconfig "$KUBECONFIG_DESTINATION2" 
+```
+
+At the end of the process, the virtual nodes are removed from the local cluster.
+
+### Uninstall Liqo
+
+Now you can uninstall Liqo from your clusters:
+
+```bash
+liqoctl uninstall
+liqoctl uninstall --kubeconfig="$KUBECONFIG_DESTINATION1" 
+liqoctl uninstall --kubeconfig="$KUBECONFIG_DESTINATION2" 
+```
+
+```{admonition} Purge
```

Comment:
I would avoid this detail about 'purge' repeated in each example.
Can we move this note into a more general place, and add a link to that page here?
(e.g., Detailed instructions about how to remove Liqo are available in THIS page).

Reply:
It could be done in another PR. In my opinion, the command examples inside the **admonition** could be removed from this example.